.eslintignore

```diff
@@ -1,5 +1,5 @@
 node_modules
-/examples/*/build
+/examples
```

Comment:
Our examples are to be run independently, we shouldn't link based on our repo's rules.

Reply:
💯